components/ProgressCard.tsx

```diff
@@ -57,14 +57,28 @@ const ProgressSVG: React.FC<Props> = ({ progressCount }) => {
 }
 
 const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  const setHeaderFromProgress = () => {
+    if (progressCount <= 13) {
+      // Has not finished 1st module yet
+      return `You're off to a great start!`
+    } else if (progressCount <= 50) {
```

Comment:
FYI, if you are returning, `else` is redundant.

Reply:
Yes, good catch.